go/vt/sqlparser/sql.y

```diff
@@ -425,6 +427,29 @@ command:
   setParseTree(yylex, nil)
 }
 
+id_or_var:
+  ID
+  {
+    $$ = NewColIdentWithAt(string($1), NoAt)
+  }
+| AT_ID
```

Comment:
This won't do anything without modifying the lexer (it know what an ID is, it would need to understand what an AT_ID is too)